packages/helper-wasm-bytecode/src/index.js

```diff
@@ -319,6 +319,13 @@ const symbolsByByte = {
   0xba: createSymbolObject("convert_u/i64", "f64"),
   0xbb: createSymbolObject("promote/f32", "f64"),
 
+  // Sign extension op proposal
+  0xc0: createSymbolObject("extend8_s", "i32"),
+  0xc1: createSymbolObject("extend16_s", "i32"),
+  0xc2: createSymbolObject("extend8_s", "i64"),
+  0xc3: createSymbolObject("extend16_s", "i64"),
+  0xc4: createSymbolObject("extend32_s", "i64"),
```

Comment:
Maybe we should change that to support two or more bytes? Atomic operations also have an additional byte.

Reply:
Are you referring to the threads proposal? i.e. https://webassembly.github.io/threads/binary/instructions.html#atomic-memory-instructions. Then it shouldn't affect this proposal but something to look out for in the future I think?